### ISSUE TITLE
Default backdrop is blocking TalkBack from accessing views below it 

### ISSUE THREAD

**Issue by mskri** [gorhom/react-native-bottom-sheet#116]
# Bug

When using a TalkBack on Android (VoiceOver seems to work okay) the default backdrop component grabs focus when navigating elements on the view. This happens even when backdrop is not visible. 

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet         | 2.0.0-alpha.7   |
| react-native                    | 0.63.2   |
| react-native-reanimated         | 1.13.1   |
| react-native-gesture-handler    | 1.8.0  |

## Steps To Reproduce

1. Use bottom sheet with the default backdrop component in a screen with other components visible on the area where the backdrop will show up.
2. Turn on TalkBack on Android and try to navigate between items on the screen by either tapping or swiping to go next/prev element. 
3. Even when backdrop is not visible TalkBack focuses on the backdrop preventing access to the views below it.

Describe what you expected to happen:

When using TalkBack to navigate the screen which has a bottom sheet with default backdrop the backdrop element does not grab focus when not visible

**Comment by HarunKilic** [comment]
This seems not to be fixed. 
I cannot access the UI even when the bottomsheet is closed. 

Packages: 
`"@gorhom/bottom-sheet": "2.0.1",`
`"react-native": "0.63.4",`
`"react-native-reanimated": "^2.0.0-rc.1",`
`"react-native-gesture-handler": "^1.9.0",`
